schedule.md

```diff
@@ -0,0 +1,89 @@
+# Session plan
+
+## Session 1: R and statistical concepts used
+
+Some of this could be pre-course reading material, see e.g. https://sbfnk.github.io/mfiidd/slides/rintro.pdf
+
+Monday 9:30-11:00
+
+- Introduction to R concepts used in the course (15 mins)
+- Introduction to statistical concepts used in the course and what they look like in stan (30 mins)
+- Practice session: probability distribution and estimates (30 mins)
+- Wrapup (15 mins)
+
+## Session 2: Delay distributions
+
+Monday 11:00-12:30
+
+- Introduction to epidemiological delays and how to represent them with probability distributions (15 mins)
+- Practice session: simulate and estimate epidemiological delays (30 mins)
+- Practice session: introducing daily censoring (30 mins)
+- Wrapup (15 mins)
```

Comment:
All the early sessions havee wrap up sections and none of the later ones do. I think its a nice thing to have but maybe all should have them and we should aim for no more than 10 minutes?